osx/KBKit/KBKit/Component/KBHelperTool.m

```diff
@@ -85,7 +85,7 @@ - (void)doInstallAlert:(KBSemVersion *)bundleVersion runningVersion:(KBSemVersio
   NSString *infoText = @"";
   if ([bundleVersion isOrderedSame:[KBSemVersion version:@"1.0.31"]]) {
     alertText = @"New Keybase feature: multiple users in macOS";
-    infoText = @"Previously, only one user of this computer could find their Keybase files at /keybase. With this update, /keybase will now support multiple users on the same computer by redirecting to keybase/ in your home directory, where most apps put files.\n\nYou may need to enter your password for this update.";  }
+    infoText = @"Previously, only one user of this computer could find their Keybase files at /keybase. With this update, /keybase will now support multiple users on the same computer by redirecting to user-specific Keybase directories in /Volumes.\n\nYou may need to enter your password for this update.";  }
```

Comment:
I suggest leaving out mention of redirecting. But will of course defer to @malgorithms.

> Previously, only one user on this computer could find their Keybase files at /keybase. With this update, /keybase will now support multiple users on the same computer! You can also access your Keybase files in /Volumes/\<you\>'s Keybase.
>
> You may need to enter your password for this update.

Reply:
I think it's great. Maybe "linking" is a more generally consumable word than "redirecting"...but no strong opinion.